packages/cysignals/patches/0001-src-cysignals-implementation.c-EMSCRIPTEN-Do-not-use.patch

```diff
@@ -0,0 +1,36 @@
+From b13d2be64ff7f855465002cd507cb830de0d43da Mon Sep 17 00:00:00 2001
+From: Matthias Koeppe <mkoeppe@math.ucdavis.edu>
+Date: Wed, 24 Jan 2024 23:24:45 -0800
+Subject: [PATCH] src/cysignals/implementation.c [EMSCRIPTEN]: Do not use
+ sigaltstack
+
+---
+ src/cysignals/implementation.c | 4 +++-
+ 1 file changed, 3 insertions(+), 1 deletion(-)
+
+diff --git a/src/cysignals/implementation.c b/src/cysignals/implementation.c
+index 2afd6ad..3088cdf 100644
+--- a/src/cysignals/implementation.c
++++ b/src/cysignals/implementation.c
+@@ -531,7 +531,7 @@ static void _sig_off_warning(const char* file, int line)
+ 
+ static void setup_alt_stack(void)
+ {
+-#if HAVE_SIGALTSTACK
++#if HAVE_SIGALTSTACK && !defined(__EMSCRIPTEN__)
```

Comment:
Could you add some more information here? What goes wrong without this patch? Compiler error? Runtime error? Should `HAVE_SIGALTSTACK` be false in Emscripten?

Reply:
As indicated in the commit message with the patch, this is https://github.com/sagemath/cysignals/pull/197, where I explain the details